### PR TITLE
Added 2 new coins.

### PR DESCRIPTION
**Pull request by globaltoken**
Hey,

I added 2 new Coins. We have Globaltoken Zhash & Globaltoken Equihash, because we use Multialgorithm.

Greetings,
Pawel

### CODE REVIEW

**Comment by Gumbees** [comment]
👍 for the looks. needs tested by one of us. can you add testnet configs so we can throw it up and test? @globaltoken

**Comment by globaltoken** [comment]
@nettts The hardfork is not released yet.
I will test it.

**Comment by sennevb** [comment]
seems cool, but node-stratum-pool needs to be merged with snomp

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
         "nonce": "^1.0.4",
         "redis": "^2.8.0",
         "request": "^2.88.0",
-        "stratum-pool": "git+https://github.com/s-nomp/node-stratum-pool.git"
+        "stratum-pool": "git+https://github.com/globaltoken/snomp-stratum-pool.git"
```
Comment (egyptianbman) [inline package.json:51]: This isn't an option. You'll need to create a PR to https://github.com/s-nomp/node-stratum-pool.git to support this coin and restore the original value.

**Review by egyptianbman — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (globaltoken) [reply package.json:51]: @egyptianbman I will restore it

**Comment by globaltoken** [comment]
Alright, everything is working with this commit. In theory Bitcoin Gold should work with this Code part as well, because it is a direct Bitcoin Fork, and not Zcash based. I tested this for Globaltoken alone and with @sennevb who integrated GLT successfully into his Pool.

The stratum pull: https://github.com/s-nomp/node-stratum-pool/pull/20

If you have any more questions, let me know about it.
Cheers,

**Comment by zzzpotato** [comment]
This appears abandoned. Closing...

**Pull request closed without merging**